Makefile

```diff
@@ -98,3 +98,30 @@ build-license-notice:
 .PHONY: generate-chart-docs
 build-chart-docs:
 	helm-docs -c charts/newrelic-prometheus-agent
+
+## Release Toolkit targets
+ifeq (, $(shell which rt))
+RT_BIN ?= go run github.com/newrelic/release-toolkit@latest
+else
+RT_BIN ?= rt
```

Comment:
I can actually remove this and if need it we can call the make with this variable

Reply:
As we fixed the go module path, I'd say remove it and just rely on the `go run`. I used this approach in this PR https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/116 for the e2e actions.

Reply:
yes, the idea of this was that if you already have a binary in your path, it would be override the go run. but I think we should always use the latest tag, and in case you want to override it , we can use do `RT_BIN=my-bin make ...`

Reply:
why latest and not v1 as the github actions?

Reply:
I haven't tried TBH but @kang-makes  mentioned that if you point to tags that are "moved" like v1 , the binary is not going to be re-fetched when it moves, but latest does

Reply:
This is a problem with local caches on our laptops: Once you have downloaded a tag it remains and go does not expect it to be changed.

Is the same behavior as when you use `docker run`: it does not redownload the image if is already present in your system (unless you explicitly add `--pull=always`).

Reply:
Oh true, we discussed it many times already and I forgot 😓